ilapfuncs.py

```diff
@@ -65,8 +67,8 @@ def datark(filefound):
     logfunc(f"Data_ark.plist function executing")
     deviceinfo()
     try:
-        os.makedirs(reportfolderbase + "Data_Ark/")
-        with open(reportfolderbase + "Data_Ark/Data Ark.html", "w") as f:
+        os.makedirs(os.path.join(reportfolderbase, "Data_Ark/"))
+        with open(os.path.join(reportfolderbase, "Data_Ark/Data Ark.html"), "w") as f:
```

Comment:
When using `os.path.join` you want to pass all paths as an arg, otherwise you're still hardcoding the separator.

Another option is to use `os.sep` and string formatting, but since we have chosen the former in ALEAPP just makes sense to follow that convention.